opam/opam-coq-archive/extra-dev/packages/gappa/gappa.dev/opam

```diff
@@ -14,6 +14,8 @@ build: [
   [ "./configure"
     # If someone knows how to ask MacPorts for the usual include and lib path, please tell me
     "CXXFLAGS=-I/opt/local/include" "LDFLAGS=-L/opt/local/lib" { os-distribution = "macports" & os = "macos" }
+    # Support installing on Apple Silicon with Homebrew
+    "CXXFLAGS=-I/opt/homebrew/include" "LDFLAGS=-L/opt/homebrew/lib" { os-distribution = "homebrew" & os = "macos" }
```

Comment:
I wonder if `/opt/homebrew` is now the default destination for homebrew - it used to be `/opt/local`.

Reply:
It is the default on Apple Silicon, see https://docs.brew.sh/Installation